### ISSUE TITLE
Not a bug just a question

### ISSUE THREAD

**Issue by HarshitRanjanRaiHDOR** [matinzd/react-native-health-connect#62]
I'm migrating project from expo to CLI, is this package is supported in CLI?

**Comment by matinzd** [comment]
It is both supported on Expo and bare RN. Our example is a bare RN project.

**Comment by HarshitRanjanRaiHDOR** [comment]
Let me rephrase the question. We have our project in React Native CLI. When we tried your package using Expo CLI it worked as expected, but while using your package in React Native CLI it doesn't event complie when we follow the same steps. And it gives the error in npm start.

**Comment by matinzd** [comment]
Same answer again 😅
Just clone the repo and go to example folder. I myself, do not even test my lib on Expo for now and I have been using RN CLI since start. If you have any problems, please upload a reproduction somewhere so I can check and also send some logs. Not compiling can be caused by various factors.

**Comment by HarshitRanjanRaiHDOR** [comment]
Let me check this.

**Comment by matinzd** [comment]
If you are getting an error when running `npm start` I can say that is not caused by this lib or any other libs and it can be some misconfigurations in your project.
